ansible/roles/images/tasks/load.yaml

```diff
@@ -0,0 +1,5 @@
+
+- name: import images from local cache
+  shell: "ctr -n k8s.io image import {{ item }}"
+  with_fileglob:
+  - "{{ images_cache }}/*"
```

Comment:
Right now, the images being generated are `tar.gz`, we should add a step in here to untar (if needed).
Similar to https://github.com/mesosphere/konvoy/blob/7d8ec7e555333cb2ef9e253aceb12f95374cc7bf/ansible/roles/packages-copy-local/tasks/main.yaml#L95-L104

Reply:
Yep. Going to add this.

Reply:
@jr0d and I made two decisions:
1. To avoid a larger than necessary cache size, we want to decompress to `ctr`'s stdin, rather than to disk.
2. We will need to use a local binary to decompress, and since `gzip` is widely available, we'll require that.